Dockerfile

```diff
@@ -1,4 +1,15 @@
-FROM rust:1.67-bullseye AS chef
+FROM rust:1.77-bullseye AS chef
+
+RUN set -x \
+    && apt-get -qq update \
+    && apt-get -qq -y install \
+    clang \
+    cmake \
+    libudev-dev \
+    libssl-dev \
+    pkg-config \
+    zlib1g-dev
```

Comment:
these look fine but ill want to review the docker file to make sure these are all required (thinking about recent `xz` situation)